tests/testthat/test-Model-class.R

```diff
@@ -853,3 +853,75 @@ test_that("MCMC computes correct values for DALogisticLogNormal model and empty
   result <- mcmc(data = data, model = model, options = options)
   expect_snapshot(result@data)
 })
+
+# TITELogisticLogNormal ----
+
+## constructor ----
+
+test_that("TITELogisticLogNormal object can be created with user constructor (linear weight)", {
+  result <- expect_silent(h_get_tite_logistic_log_normal("linear"))
+  expect_valid(result, "TITELogisticLogNormal")
+})
+
+test_that("TITELogisticLogNormal object can be created with user constructor (adaptive weight)", {
+  result <- expect_silent(h_get_tite_logistic_log_normal("adaptive"))
+  expect_valid(result, "TITELogisticLogNormal")
+})
+
+## mcmc ----
+
+test_that("MCMC computes correct values for TITELogisticLogNormal model (linear)", {
+  data <- h_get_data_da()
+  model <- h_get_tite_logistic_log_normal("linear")
+  options <- h_get_mcmc_options(small = TRUE, fixed = TRUE)
+
+  result <- mcmc(data = data, model = model, options = options)
+  expect_snapshot(result@data)
+})
+
+test_that("MCMC computes correct values for TITELogisticLogNormal model (linear, data 2)", {
+  data <- h_get_data_da_2()
+  model <- h_get_tite_logistic_log_normal("linear")
+  options <- h_get_mcmc_options(small = TRUE, fixed = TRUE)
+
+  result <- mcmc(data = data, model = model, options = options)
+  expect_snapshot(result@data)
+})
+
+test_that("MCMC computes correct values for TITELogisticLogNormal model (linear, empty data)", {
+  data <- h_get_data_da(empty = TRUE)
+  model <- h_get_tite_logistic_log_normal("linear")
+  options <- h_get_mcmc_options(small = TRUE, fixed = TRUE)
+
+  # Suppress JAGS warnings about unused variables `zeros` and `cadj`.
```

Comment:
Hi @danielinteractive - I will open a separate issue for this, as it requires some global changes. This "unused variables" JAGS warning is because `model@modelspecs` returns all of the parameters, regardless of the whether the (merged `datamodel`, `priormodel`) JAGS model use it or not. And in this case, the `priormodel` dose not use `zeros` and `cadj` vars. Probably we may need to parametrize `model@modelspecs` function with some additional argument, such as `from_prior`.

Reply:
thx @wwojciech , sounds good - either reparametrize or similar as with the `datanames` add another slot that could be used if needed only? we can discuss then more in the other issue.